images/build-e2e/README.md

```diff
@@ -6,41 +6,42 @@ The container connects through ssh to the target host and copy the right binary
 
 ## Envs
 
-* PLATFORM              define target platform (windows, macos, linux).  
-* TARGET_HOST           dns or ip for the target host.  
-* TARGET_HOST_USERNAME  username for target host.  
-* TARGET_HOST_KEY_PATH  private key for user. (Mandatory if not TARGET_HOST_PASSWORD).  
-* TARGET_HOST_PASSWORD  password for user. (Mandatory if not TARGET_HOST_KEY_PATH).  
-* PULL_SECRET_FILE_PATH pull secret file path (local to container).  
-* BUNDLE_VERSION        testing agaisnt crc released version bundle version for crc released version. (Mandatory if not BUNDLE_LOCATION).  
-* BUNDLE_LOCATION       when testing crc with custom bundle set the bundle location on target server. (Mandatory if not BUNDLE_VERSION).  
-* RESULTS_PATH          Optional. Path inside container to pick results and logs from e2e execution.  
+**PLATFORM**:*define target platform (windows, macos, linux).*
+**TARGET_HOST**:*dns or ip for the target host.*  
+**TARGET_HOST_USERNAME**:*username for target host.*  
+**TARGET_HOST_KEY_PATH**:*private key for user. (Mandatory if not TARGET_HOST_PASSWORD).*  
+**TARGET_HOST_PASSWORD**:*password for user. (Mandatory if not TARGET_HOST_KEY_PATH).*  
+**PULL_SECRET_FILE_PATH** pull secret file path (local to container).*  
+**BUNDLE_VERSION**:*(Mandatory if not BUNDLE_LOCATION). Testing agaisnt crc released version bundle version for crc released version.*
+**BUNDLE_LOCATION**:*(Mandatory if not BUNDLE_VERSION). When testing crc with custom bundle set the bundle location on target server.*  
+**RESULTS_PATH**:*(Optional). Path inside container to pick results and logs from e2e execution.*
```

Comment:
Fwiw, I'd do this reformatting, and the white space changes below in a separate commit so that it's easier to see the`CLEANUP_HOME`-related changes.

Reply:
Changed and splitted